docs/release-notes/v11.3.0.md

```diff
@@ -0,0 +1,159 @@
+---
+id: v11.3.0
+title: Version v11.3.0 Release Notes
+sidebar_label: v11.3.0
+image: "img/release-note-link-preview.png"
+---
+
+_June 22th, 2023_
+
+## Console
+
+### New Features
+
+#### Manage and configure Webhooks directly via Console UI
+
+Starting from today, it will be possible to view, create, edit and delete custom Console Webhooks directly through the user interface.
+Whereas before today this was only possible through Console CMS, now the user can enter the "Company Overview" area and under the "Webhooks" section can easily and intuitively manage all the respective hooks.
+
+[da inserire link doc]
+
+#### Backoffice Configurator
+
+Starting from this new version the Backoffice Configurator provides some new features listed below!
+
+- Shared Property page is available, allowing to define shared properties and interact with them.
+For more information, [go to the documentation](/business_suite/backoffice-configurator/compose-pages#shared-properties)
+- `dynamic-icon` manifest schema hint allows the dynamic selection of icons
+- `mia/endpoints/data-source` manifest schema hint allows selecting single-view, mongo-view, and projections endpoints in no code interface
+- Field `entry` of Micro Frontend pages can be created through the no-code interface
+- Support for external assets via configuration of a service worker which acts as a reverse proxy is available.
+For more information, [go to the documentation](/business_suite/backoffice-configurator/external-components/overview).
+
+### Improvements
+
+#### Design Overview
+
+We have updated the Design Overview section with the following features:
+- CRUDs tab becomes **Data Models** and includes also Single Views and Projections collections
+- **contains** operator to allow case insensitive free search on text fields
+- **not set** operator to filter null values
+- fix graphic issues
+
+For more information, [go to the documentation](/development_suite/governance/design-overview.md).
+
+![Design Overview](./img/overview/design-overview-update.png)
+
+#### Backoffice Configurator
+
+Some improvements on the Backoffice Configurator provided with this version are listed below!
+
+- Select input in the components props editing menu can now be cleared
+- Preview Status bar filters out info notifications from count total
+- Primary color is chosen for superscripts of badges (Variables and Service-Worker buttons)
+- Configuration parsing errors are now notified and no longer break the editor
+- Test variables are trimmed on the creation
+
+### Bug Fix
+
+#### Backoffice Configurator
+
+This version addressed few bugs listed below.
+
+- Values of properties with the same name in different components are no longer bound
+- Fixed references resolution in the preview
+- When the compose application is removed, its metadata are deleted
+
+## Fast Data
+
+### New Features
+
+#### Fast Data No Code Configuration for Aggregation
```

Comment:
@ThisIsDemetrio @fedepini can we add a screenshot here?

Reply:
@TheRusCompass done!